extensions/core/interfaces/stock/interface.vue

```diff
@@ -0,0 +1,335 @@
+        .then(function (response) {
+
+          vm.showChart = true;
+          vm.message = "";
+
+          vm.companyName = response.data.quote.companyName;
+          vm.primaryExchange = response.data.quote.primaryExchange;
+          vm.latestPrice = response.data.quote.latestPrice.toFixed(2);
+
+          vm.details.open = response.data.quote.open.toFixed(2);
+          vm.details.marketCap = numAbbr.abbreviate(response.data.quote.marketCap, 2).toUpperCase();
+          vm.details.high = response.data.quote.high.toFixed(2);
+          vm.details.low = response.data.quote.low.toFixed(2);
+          vm.details.week52High = response.data.quote.week52High.toFixed(2);
+          vm.details.week52Low = response.data.quote.week52Low.toFixed(2);
+
+          let dataLabels = [];
+          let dataValues = [];
+
+          for(var key in response.data.chart.data) {
+              if(response.data.chart.data.hasOwnProperty(key)) {
+                  dataLabels[key] = response.data.chart.data[key].date;
+                  dataValues[key] = response.data.chart.data[key].close;
+              }
+          }
+
+          vm.historyData.labels = dataLabels;
+          vm.historyData.datasets[0].data = dataValues;
+
+          if(response.data.quote.change > 0){
+            vm.trending = "trending_up";
+            vm.change = "+" + vm.$lodash.round(response.data.quote.change, 2);
+            vm.changePercent = "+" + vm.$lodash.round(response.data.quote.changePercent*100, 2) + "%";
+            vm.historyData.datasets[0].backgroundColor = hexToRgba(getCSS("--green"), 0.2);
+            vm.historyData.datasets[0].borderColor = hexToRgba(getCSS("--green"), 1.0);
+          } else {
+            vm.trending = "trending_down";
+            vm.change = vm.$lodash.round(response.data.quote.change, 2);
+            vm.changePercent = vm.$lodash.round(response.data.quote.changePercent*100, 2) + "%";
+            vm.historyData.datasets[0].backgroundColor = hexToRgba(getCSS("--red"), 0.2);
+            vm.historyData.datasets[0].borderColor = hexToRgba(getCSS("--red"), 1.0);
+          }
+
+          // Generate History Chart
+          var ctx = document.getElementById("history");
+          var history = new Chart(ctx, {
+              type: 'line',
+              data: vm.historyData,
+              options: vm.historyOptions
+          });
+        })
+        .catch(function (error) {
+          vm.message = 'Invalid stock symbol';
+          vm.showChart = false;
+          vm.latestPrice = "0.00";
+          vm.change = "0.00";
+          vm.changePercent = "0.00%";
+          vm.trending = "trending_flat";
+          console.log("Error:", error);
+        })
+    }
+  }
+};
+</script>
+
+<style lang="scss" scoped>
+.v-input {
+  max-width: var(--width-x-small);
+  display: inline-block;
+}
+.v-input input {
+  text-transform: uppercase;
+  // Why can't I style this input?????
```

Comment:
Styling is scoped to the current component. Everything inside the `v-input` component is a level deeper and therefore can't be reached from the parent. 

There _is_ a way to override te scope: `.v-input /deep/ input` but I highly recommend not using it, as it defeats the whole purpose of using scoped styles in the first place. Instead, consider adding a prop to the `v-input` component that adds this styles in the component itself.